public/components/ContentPanel/ContentPanel.tsx

```diff
@@ -46,13 +51,30 @@ const ContentPanel = (props: ContentPanelProps) => (
       alignItems="center"
     >
       <EuiFlexItem>
-        <EuiTitle
-          size={props.titleSize || 'l'}
-          className={props.titleClassName || ''}
-        >
-          <h3>{props.title}</h3>
-        </EuiTitle>
+        <EuiFlexGroup justifyContent="flexStart" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiTitle
+              size={props.titleSize || 'l'}
+              className={props.titleClassName || ''}
```

Comment:
Can we set the title's default className as `preview-title` ? I see most parts use this class. We can avoid some extra work by doing this. 

Similar for subTitle class, 
```
.content-panel-subtitle {
  font-size: 12pt;
  color: #69707D;
}
```

Reply:
Sure. will change default className to `preview-title`, and add 'content-panel-subtitle' as default className